src/app/joint-control/joint-control-core/motor-position/motor-position.component.css

```diff
@@ -1,12 +1,7 @@
-.right-icon-label {
-    color: white;
-    text-align: left;
-    font-size: 18px;
-}
-
```

Comment:
This can not be replaced because we need it to display 
![image](https://github.com/pib-rocks/cerebra/assets/144774775/32b59d84-10b6-41ba-ab54-7b5eeb763547)